Simplified/Audiobooks/DPLA/JWKResponse.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  JWKResponse.swift
+//  Simplified
+//
+//  Created by Vladimir Fedorov on 31.08.2020.
+//  Copyright © 2020 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+import NYPLAudiobookToolkit
+
+struct JWKResponse: Codable {
+  let keys: [JWK]
+}
+
+struct JWK: Codable {
+  /// Custom JWK structure in response
+  private enum JWKKeys: String, CodingKey {
+    case kty
+    case publickKeyEncoded = "http://www.feedbooks.com/audiobooks/signature/pem-key"
+  }
+  /// Key type
+  let kty: String
```

Comment:
same thing here, then there's no need for comments either.

Reply:
removed `kty`, not in use anymore